saleor/graphql/account/mutations/base.py

```diff
@@ -100,7 +100,7 @@ def _set_password_for_user(cls, email, password, token):
     def handle_typed_errors(cls, errors: list):
         account_errors = [
             AccountError(field=e.field, message=e.message, code=code)
-            for e, code in errors
+            for e, code, _ in errors
```

Comment:
```suggestion
            for e, code, _params in errors
```

This would help us know is the third one without triggering warnings from the linters.